src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx

```diff
@@ -35,9 +44,9 @@ const BookingPill = styled.div<{ disabled: boolean; altStyle: boolean; color: st
   `}
 `
 
-const BookingStatus = styled.span`
+const BookingStatus = styled.span<{ altStyle: boolean }>`
```

Comment:
nit: Not loving the name `altStyle`. It gives no context to what it actually is.
Perhaps `large: boolean`, seeing as it makes the font larger? Not sure.

Reply:
Same applies to the above `BookingPill` component